src/Symfony/Component/Form/Flow/AbstractFlowButtonType.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Flow;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Flow\Type\FlowButtonType;
+
+/**
+ * @author Yonel Ceruto <open@yceruto.dev>
+ */
+abstract class AbstractFlowButtonType extends AbstractType implements FlowButtonTypeInterface
```

Comment:
should this class be in the Type namespace?
also, what about naming it AbstractButtonFlowType

Reply:
I've mirrored the same structure as other form classes, for example, all existing related-type abstracts and interfaces are located directly under the `Form/` directory. Here, they are directly under `Form/Flow/`. 

Is it okay to keep it like this for consistency?

Reply:
Makes sense!